docs/css/font-size.md

```diff
@@ -0,0 +1,84 @@
+> Typography
+
+# Font Size
+Utilities for controlling the font size of an element.
+
+## Quick reference
+
+<qr-table />
+
+## Basic usage
+
+### Setting the font size
+Control the font size of an element using the `text-{size}` utilities.
+These come with built-in line-heights.
+Line-heights can be overridden by the `.leading-{size}` classes, but please only do that as a last resort since that might take them out of the brand theming patterns.
+
+<example-container>
+  <div class="ex-inner-box max-w-full s-text">
+    <strong class="text-xs">text-xs</strong>
+    <p class="text-xs">The quick warp scientist</p>
+    <strong class="text-s">text-s</strong>
+    <p class="text-s">The quick warp scientist</p>
+    <strong class="text-m">text-m</strong>
+    <p class="text-m">The quick warp scientist</p>
+    <strong class="text-ml">text-ml</strong>
+    <p class="text-ml">The quick warp scientist</p>
+    <strong class="text-l">text-l</strong>
+    <p class="text-l">The quick warp scientist</p>
+    <strong class="text-xl">text-xl</strong>
+    <p class="text-xl">The quick warp scientist</p>
+    <strong class="text-xxl">text-xxl</strong>
+    <p class="text-xxl">The quick warp scientist</p>
+    <strong class="text-xxxl">text-xxxl</strong>
+    <p class="text-xxxl">The quick warp scientist</p>
+  </div>
+</example-container>
+
+```html
+<p class="text-xs ...">The quick warp scientist ...</p>
+<p class="text-s ...">The quick warp scientist ...</p>
+<p class="text-m ...">The quick warp scientist ...</p>
+<p class="text-ml ...">The quick warp scientist ...</p>
+<p class="text-l ...">The quick warp scientist ...</p>
+<p class="text-xl ...">The quick warp scientist ...</p>
+<p class="text-xxl ...">The quick warp scientist ...</p>
+<p class="text-xxxl ...">The quick warp scientist ...</p>
+```
+
+### Setting the headings
+
+Control the heading sizes using the `h1` - `h5` utilities. These classes aim to contain all the styling needed for each heading-level pr. brand.
```

Comment:
I think we should advise using (and use) the same classes Figma is exposing, which is `t1-t5`. h1-h5 is something that came with Fabric and we made shortcuts in Warp that would map those to t1-t5, which generate the same style.